_posts/2018-09-09-eager-style-transfer/eager-style-transfer.Rmd

```diff
@@ -0,0 +1,443 @@
+---
+title: "Neural style transfer with eager execution and Keras"
+description: |
+  Continuing our series on combining Keras with TensorFlow eager execution, we show how to implement neural style transfer in a straightforward way. Based on this easy-to-adapt example, you can easily perform style transfer on your own images. 
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: http://www.rstudio.com/
+bibliography: bibliography.bib
+date: 09-09-2018
+categories:
+  - Neural style transfer
+  - Keras
+  - Eager
+output:
+  radix::radix_article:
+    self_contained: false
+preview: images/style_epoch_1000.png
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+How would your summer holiday's photos look had Edvard Munch painted them? (Perhaps it's better not to know).
+Let's take a more comforting example: How would a nice, summarly river landscape look if painted by Katsushika Hokusai?
+
+Style transfer on images is not new, but got a boost when [@GatysEB15a] showed how to successfully do it with deep learning.
+The main idea is straightforward: Create a hybrid that is a tradeoff between the _content image_ we want to manipulate, and a _style image_ we want to imitate, by optimizing for maximal resemblance to both at the same time.
+
+If you've read the chapter on neural style transfer from [Deep Learning with R](https://tensorflow.rstudio.com/learn/resources.html), you may recognize some of the code snippets that follow.
+However, there is an important difference: This post uses TensorFlow eager execution, allowing for an imperative way of coding that makes it easy to map concepts to code.
+Just like previous posts on eager execution on this blog, this is a port of a [Google Colaboratory notebook](https://colab.research.google.com/github/tensorflow/models/blob/master/research/nst_blogpost/4_Neural_Style_Transfer_with_Eager_Execution.ipynb) that performs the same task in Python.
+
+As usual, please make sure you have the required package versions installed. And no need to copy the snippets - you'll find the complete code among the [Keras examples](https://github.com/rstudio/keras/tree/master/vignettes/examples/eager-styletransfer.R).
+
+
+## Prerequisites
+
+The code in this post depends on the development versions of several of the TensorFlow R packages. You can install these packages as follows:
```

Comment:
I think we can change this to CRAN assuming we don't add any new functionality based on reviewing the post.